src/html/components/concept.html

```diff
@@ -4,4 +4,36 @@ <h2 class="section-title">The Concept</h2>
   <p class="section-intro">Meet Florence</p>
   <p>Our explorations led to the concept of Florence — a personal companion that would help patients understand and manage their medication regimen.  A chatbot at heart, patients would converse with her through text and picture messages.</p>
   <p>If developed, a simple, familiar interface would hide her powerful intelligence. Behind the scenes, machine learning and behavioral modeling would enable her to offer a tailored, human experience based on each patient’s needs.</p>
+  <div class="box demo demo-one">
+    <img src="images/section1-illo.svg" alt="" class="illo cup">
+    <div class="explain">
+      <h3>She gets to know you.</h3>
+      <p>Florence strikes up a conversation, learning details about your life, health, goals, and preferences in an engaging, human way.</p>
+      <p>This conversation unfolds over time, making this feel less like twenty questions and more like the start of a relationship. </p>
+    </div>
+    <div class="messages">
+      <ul>
+        <li class="messages-left">Hi! I'm Florence. What's your name?</li>
+        <li class="messages-right">Mike</li>
+        <li class="messages-left">Nice to meet you, Mike. I can help you set up your account with a few simple questions.</li>
+        <li class="messages-left">You can answer these now or whenever you like. I'm always here!</li>
+        <li class="messages-left recent">First question: how old are you?</li>
+        <li class="messages-right">65</li>
```

Comment:
I'd make these li classes singular, e.g. `message-right`. Other options would be `m-left`/`m-right` or `left`/`right` or `from`/`to`.